packages/redrip/src/redripify/use-map-animate-to-style.ts

```diff
@@ -297,21 +297,21 @@ export default function useMapAnimateToStyle<Animate>({
 
       if (isColor(key)) {
         // TODO: FIX THIS
-        if (__DEV__) {
-          if (
-            typeof value === 'string' &&
-            !value.startsWith('rgb') &&
-            !value.startsWith('#')
-          ) {
-            console.error(
-              `[${PackageName}]: You passed ${key}: ${value}, but not all color values are supported yet in Reanimated 2. ☹️ 
+        // if (__DEV__) {
```

Comment:
I think this is worth adding an issue in reanimated's repo. I can reproduce it and do that later.